app/services/pro_memberships/expiration_notifier.rb

```diff
@@ -0,0 +1,47 @@
+module ProMemberships
+  # Notifies Pro members with insufficient credits of the impeding expiration
+  class ExpirationNotifier
```

Comment:
This is called by two rake tasks: one that checks if there are expiring memberships 1 week in the future and one for expiring memberships 1 day in the future